action.yml

```
@@ -34,10 +34,16 @@ inputs:
       or valid Ansible directories according to the Ansible role
       directory structure.
     required: true
+  override-deps:
+    description: |
```

Should we also add a default?
```suggestion
    default: ''
    description: |
```